### PR TITLE
Improve Simplified Chinese translations

### PR DESCRIPTION
**Pull request by contactalexliu**
* **Please check if the PR fulfills these requirements**
- [x] The changes have been tested locally
- [x] There are no breaking changes

* **What kind of change does this PR introduce?**
Some Simplified Chinese translations were improved to be more in line with the vernacular.

### CODE REVIEW

File: Translations/translation_ZH_CN.json [hunk 1]
```diff
@@ -4,27 +4,27 @@
...
     "OffString": "关"
   },
   "messagesWarn": {
     "ResetOKMessage": "已重置！",
-    "SettingsResetMessage": "设置已被重置！",
-    "NoAccelerometerMessage": ["No accelerometer", "detected!"],
-    "NoPowerDeliveryMessage": ["No USB-PD IC", "detected!"],
+    "SettingsResetMessage": "设定已被重置！",
+    "NoAccelerometerMessage": "未感应加速度计！",
+    "NoPowerDeliveryMessage": "未感应PD电路！",
```
Comment (alvinhochun) [inline Translations/translation_ZH_CN.json:25]: Would "檢測" be more suitable?

File: Translations/translation_ZH_CN.json [hunk 2]
```diff
@@ -63,146 +63,146 @@
...
     "TempChangeShortStep": {
-      "text2": "温度调整 短",
+      "text2": "短按温度调整",
       "desc": "调校温度时短按一下的温度变幅"
     },
     "TempChangeLongStep": {
-      "text2": "温度调整 长",
-      "desc": "调校温度时长按按键的温度变幅"
+      "text2": "长按温度调整",
+      "desc": "调校温度时长按一下的温度变幅"
```
Comment (alvinhochun) [inline Translations/translation_ZH_CN.json:161]: You can long press and hold to adjust by multiple steps with autorepeat, so "按一下" isn't quite accurate for this setting.

**Review by alvinhochun — COMMENTED** [review]
Not sure about the changes to the detailed screen, but if you think that's cooler then sure 😅 

Added some inline comments that you might want to consider, otherwise seems good to me. Would you like to keep maintaining the Simplified Chinese translations in the future?

**Comment by contactalexliu** [comment]
> Not sure about the changes to the detailed screen, but if you think that's cooler then sure 😅

I was inspired by the translations for these strings in some other languages to try and find a more graphical way to represent everything but without using CJK characters 😆

> Added some inline comments that you might want to consider, otherwise seems good to me. Would you like to keep maintaining the Simplified Chinese translations in the future?

As I am still learning, I look forward to working with you on them; I will do my best! ✊

File: Translations/translation_ZH_CN.json (hunk 1 above)
Comment (contactalexliu) [reply Translations/translation_ZH_CN.json]: Absolutely. I've implemented your suggestion in the last commit as "未检测到".

File: Translations/translation_ZH_CN.json (hunk 2 above)
Comment (contactalexliu) [reply Translations/translation_ZH_CN.json]: Good point. I've changed both descriptions in the last commit to say "按键" for consistency.

Comment (alvinhochun) [reply Translations/translation_ZH_CN.json]: Sure. I would tend towards clarity over consistency, but either way is clear enough. It's mostly down to personal preferences.

**Comment by Ralim** [comment]
@contactalexliu Just wanted to check if you are happy with this as it stands?

**Comment by alvinhochun** [comment]
Sure, please merge.

**Pull request merged**